README.md

```diff
@@ -8,10 +8,16 @@ Swift 2.2 or later is required.
 
 ## Example Table
 
+SQLite file is here.
+
+```bash
+sqlite3 Database/main.sqlite
```

Comment:
I think this should be more of where the sqlite file is stored instead of a bash command.